### ISSUE TITLE
No onRest or equivelant event

### ISSUE THREAD

**Issue by mattwilliams85** [react-spring/react-use-gesture#191]
Hey, just wanted to save I love this library but have an encountered a specific scenario I can't seem to find a solution for.

I have a list of "draggable" items in a list.
When using `onDrag`, I'm setting the z-index on the selected element to be higher than the other list items so it will always be on top as it moves. The issue is that when I let go of the element, I need to keep that higher z-index until it's at rest, otherwise it can potentially be overlapped as it travels back to it's starting position.

`onDragEnd` only triggers on mouse up, but not actually the animation's resting point. The react-spring `onRest` method only seems to trigger at the start of the event. Is there any known solution for this problem?

...
tldr; There doesn't seem to be a `onRest` hook or any way to listen to the end of the full drag animation.

**Comment by dbismut** [comment]
Hey, I've seen that you closed the issue. It's true that this is more of a challenge related to react-spring, but feel free to expose your solution here!
